src/api/resources/mixins.py

```diff
@@ -244,4 +244,5 @@ def edit(self, request, project_pk, pk):
             )
             return Response({"id": str(collect_record.pk)})
         except Exception as err:
+            print(err)
```

Comment:
Still needed?

Reply:
wanted to keep that in there for logging